packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -137,7 +146,10 @@ class CallContext {
       /* @conditional-compile-remove(unsupported-browser) */ environmentInfo: clientState.environmentInfo,
       /* @conditional-compile-remove(unsupported-browser) */ unsupportedBrowserVersionsAllowed: false,
       /* @conditional-compile-remove(rooms) */ roleHint: options?.roleHint,
-      /* @conditional-compile-remove(video-background-effects) */ videoBackgroundImages: options?.videoBackgroundImages,
+      /* @conditional-compile-remove(video-background-effects) */ videoBackgroundImages:
+        options?.videoBackgroundOptions?.videoBackgroundImages,
+      /* @conditional-compile-remove(video-background-effects) */
```

Comment:
I am thinking we should add another feature flag as "in-progress" feature other than "video-background-effects", so we can control lazy loading shipping timeframe into beta release independently

Reply:
They have to ship at the same time though no? (Unless a new feature creeps in that wants lazy-loading sooner but that feels unlikely)

Reply:
Yeah, we are shipping the dependency isolation along with video effects, so I think we can keep these under same CC